poseidon/src/parameters.rs

```diff
@@ -0,0 +1,37 @@
+use ark_bls12_381::Fr;
+use ark_sponge::poseidon::PoseidonParameters as ArkSpongePoseidonParameters;
+use once_cell::sync::Lazy;
+use poseidon_paramgen::{Alpha, PoseidonParameters};
+
+// Poseidon parameters generated for the Fr (Fp256) finite field
+pub mod fr_parameters {
+    use ark_std::vec;
+    include!(concat!(env!("OUT_DIR"), "/parameters.rs"));
```

Comment:
This looks suspicious. Maybe add some dir `parameters` or `resources` and assume parameters are there.

Reply:
Here's the file generated by build.rs in my env, using just the defaults:
![2023-01-11-1673447495_screenshot_1920x1080](https://user-images.githubusercontent.com/468572/211832632-c426ba7e-1c52-4f75-8938-3c3b1fdbec9a.jpg)

Reply:
Let's keep the discussion in one thread, resolving this one.